### PR TITLE
Add and remove

### PR DESCRIPTION
**Pull request by Osoro254Alex**
I did the following on Add and Remove branch:

- Created a new Branch on To-Do-List Repo.
- Modify the html file.
- Modify index.js file.
- Modify the css file.
- Fix Linter Error.
- Push to remote.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,43 +1,63 @@
...
+    if (mainInput.value !== '') {
+      const objects = objCreater();
+      loadList(objects);
+      list.push(objects);
+      list.sort((a, b) => a.index - b.index);
+      storeArray(list);
+      e.preventDefault();
+    }
+  }
+});
```
Comment (omarsalem7) [inline src/index.js]: - [ ] Kindly consider when adding a todo and after refreshing, everything has vanished and your data is saved in localStorage and get from it.
- [ ] Also consider when edit todo is saved in local storage and displayed on the screen.
- [ ] Also consider when deleting a todo to update the index for every todo in the list

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @Osoro254Alex 😊, 

Good job so far! 👏
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

## Highlights
- There is no linter error present ✔️
- The Pull Request has a proper title and description. ✔️

## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Osoro254Alex** [comment]
@omarsalem7 
## Changes Implemented 

- when adding a todo and after refreshing everything Should remain .
- when edit todo is saved in local storage and displayed on the screen.
- update the index for every todo in the list.

**Review by addod19 — APPROVED** [review]
### Hi @Osoro254Alex, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### Highlights
- [ ] Local storage implemented :100: for add and edit actions :rocket: 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Osoro254Alex** [comment]
@addod19 

- Checked changes Implemented
- Approved

**Pull request merged**